packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -197,13 +199,14 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
     this.toolbarTextStyle,
     this.titleTextStyle,
     this.systemOverlayStyle,
-  }) : assert(automaticallyImplyLeading != null),
-       assert(elevation == null || elevation >= 0.0),
-       assert(primary != null),
-       assert(toolbarOpacity != null),
-       assert(bottomOpacity != null),
-       preferredSize = Size.fromHeight(toolbarHeight ?? kToolbarHeight + (bottom?.preferredSize.height ?? 0.0)),
-       super(key: key);
+  })  : assert(automaticallyImplyLeading != null),
```

Comment:
It looks like a bunch of unnecessary reformatting happened (dartfmt?); need to revert that.

Reply:
Should be fixed now. This accidentally happened while I was resolving conflicts.

Reply:
Looks good although GitHub thinks that you need to resolve merge conflicts in app_bart.dart

Reply:
Presumably caused by https://github.com/flutter/flutter/pull/80380, will fix

Reply:
Thanks! Will land this when the build tree is green again.